spec/requests/api/v1/participants_spec.rb

```diff
@@ -8,13 +8,17 @@
     let(:cpd_lead_provider) { create(:cpd_lead_provider, lead_provider: lead_provider) }
     let(:lead_provider) { create(:lead_provider) }
     let(:partnership) { create(:partnership, lead_provider: lead_provider) }
+    let(:school_cohort) { create(:school_cohort, school: partnership.school, cohort: partnership.cohort, induction_programme_choice: "full_induction_programme") }
     let(:token) { LeadProviderApiToken.create_with_random_token!(cpd_lead_provider: cpd_lead_provider) }
     let(:bearer_token) { "Bearer #{token}" }
 
     before :each do
       mentor_profile = create(:mentor_profile, school: partnership.school, cohort: partnership.cohort)
-      create_list :early_career_teacher_profile, 2, mentor_profile: mentor_profile, school: partnership.school, cohort: partnership.cohort
+      create_list :early_career_teacher_profile, 2, mentor_profile: mentor_profile, school_cohort: school_cohort
+      ect_teacher_profile = ParticipantProfile::ECT.first.teacher_profile
+      create(:participant_profile, :withdrawn, :ect, teacher_profile: ect_teacher_profile, school_cohort: school_cohort)
     end
+    let!(:withdrawn_ect_profile) { create(:participant_profile, :withdrawn, :ect, school_cohort: school_cohort) }
```

Comment:
So we have two withdrawn ect profiles? One for fresh user, one for a user that has another, active participant profile?

Reply:
I wonder if there is a way to make it more obvious to the reader of the test.

Reply:
Quite a hard thing to name. will try